packages/react-native-reanimated/RNReanimated.podspec

```diff
@@ -8,7 +8,6 @@ assert_minimal_react_native_version($config)
 $new_arch_enabled = ENV['RCT_NEW_ARCH_ENABLED'] != '0'
 assert_new_architecture_enabled($new_arch_enabled)
 
-folly_flags = "-DFOLLY_NO_CONFIG -DFOLLY_MOBILE=1 -DFOLLY_USE_LIBCPP=1 -Wno-comma -Wno-shorten-64-to-32"
```

Comment:
TODO: add backward-compatibility

Reply:
I've tested this back on React Native 0.76 and it still works so I assume this is already backward-compatible.